### PR TITLE
extend kata

### PR DESCRIPTION
**Pull request by davidbecker**

### CODE REVIEW

File: alphabet-cipher/test/alphabet_cipher/coder_test.clj [hunk 1]
```diff
@@ -15,3 +15,10 @@
            (decode "vigilance" "hmkbxebpxpmyllyrxiiqtoltfgzzv")))
     (is (= "meetmebythetree"
            (decode "scones" "egsgqwtahuiljgs")))))
+
+(deftest test-decypher
+  (testing "can extract the secret keyword given an cyrpted message and tho original message"
```
Comment (gigasquid) [inline alphabet-cipher/test/alphabet_cipher/coder_test.clj]: small typo "tho" instead of "the"

**Comment by gigasquid** [comment]
Very cool.  Do you have a solution that you would like to link to in the README for people who get stuck?

**Comment by davidbecker** [comment]
I haven't started the kata, so I don't have a working solution. Maybe you leave the pull request open until I have one?

**Comment by gigasquid** [comment]
sure

**Comment by davidbecker** [comment]
I found a little time over the weekend and got the tests to pass :)

**Comment by gigasquid** [comment]
Thanks!

**Pull request merged**